### PR TITLE
configure gcc@8 with --disable-nls

### PR DESCRIPTION
**Pull request by MatthewFluet**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ ] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

Closes Homebrew/linuxbrew-core#13240.

### CODE REVIEW

File: Formula/gcc@8.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ class GccAT8 < Formula
   url "https://ftp.gnu.org/gnu/gcc/gcc-8.3.0/gcc-8.3.0.tar.xz"
   mirror "https://ftpmirror.gnu.org/gcc/gcc-8.3.0/gcc-8.3.0.tar.xz"
   sha256 "64baadfe6cc0f4947a84cb12d7f0dfaf45bb58b7e92461639596c21e02d97d2c"
+  revision 1
```
Comment (iMichka) [inline Formula/gcc@8.rb]: `revision 1 unless OS.mac?`

File: Formula/gcc@8.rb [hunk 2]
```diff
@@ -110,6 +111,7 @@ def install
       "--with-isl=#{Formula["isl"].opt_prefix}",
       "--enable-checking=release",
       "--with-pkgversion=Homebrew GCC #{pkg_version} #{build.used_options*" "}".strip,
+      "--disable-nls",
```
Comment (iMichka) [inline Formula/gcc@8.rb]: Can you move this a few lines up? And also the `--enable-checking=release` line.

We should have something like this:
```
--prefix=#{prefix}
--libdir=#{lib}/gcc/#{version_suffix}
--disable-nls
--enable-checking=release
--enable-languages=#{languages.join(",")}
--program-suffix=-#{version_suffix}
```

This aligns the code with the gcc@8 formula in homebrew-core.

**Comment by MatthewFluet** [comment]
I've made all of the requested changes.  The failure on CircleCI is:
```
==> brew bottle --merge --write --no-commit ./gcc@8--8.3.0_1.x86_64_linux.bottle.json --keep-old
Error: --keep-old was passed but there are changes in:
cellar: old: ":any", new: "/home/linuxbrew/.linuxbrew/Cellar"
==> gcc@8
==> FAILED
```
I'm not sure whether or not that is expected.

**Comment by iMichka** [comment]
Right, this is an annoying CI bug we have for some formulae. Can you also remove the `cellar :any` line?

**Comment by MatthewFluet** [comment]
Removing `cellar: any` allowed both checks to pass.

**Comment by iMichka** [comment]
Thanks @MatthewFluet !

**Pull request closed without merging**